src/ncp/ncp_base_mtd.cpp

```diff
@@ -4589,8 +4612,8 @@ void NcpBase::ProcessThreadChangedFlags(void)
         otChangedFlags    mThreadFlag;
         spinel_prop_key_t mPropKey;
     } kFlags[] = {
-        {OT_CHANGED_IP6_ADDRESS_ADDED, SPINEL_PROP_IPV6_ADDRESS_TABLE},
-        {OT_CHANGED_IP6_ADDRESS_REMOVED, SPINEL_PROP_IPV6_ADDRESS_TABLE},
+        {OT_CHANGED_IP6_ADDRESS_ADDED, SPINEL_PROP_IPV6_UNICAST_ADDRESS_TABLE},
+        {OT_CHANGED_IP6_ADDRESS_REMOVED, SPINEL_PROP_IPV6_UNICAST_ADDRESS_TABLE},
```

Comment:
Suggest keeping the old one and maybe add new property as well?

Reply:
SG! Updated.

Reply:
I found that this is not working because one flag can only maps to one property change. To keep the old one, we need to add a new value for `otChangedFlags`. However it's currently 32-bit and they are all used. We'll need to wait until this is extended to 64 bits and add a new flag. I'll just use the existing `SPINEL_PROP_IPV6_ADDRESS_TABLE` for now.